adventuregame.rb

```diff
@@ -1,73 +1,109 @@
+# frozen_string_literal: true
+
+# == Adventure \Game
+#
+# This is gonna be such a great game! Here's some details...
+# - TODO: add details
+#
+# == Running the game
+#
+# To run the game, ... (TODO: moar details needed; etc.)
+
```

Comment:
...at the opening of a file (up here, at the top), should be some explanation of _THIS_ file and what it's for in general...